src/components/findTheQuarterPage/FindTheQuarterPage.js

```diff
@@ -0,0 +1,84 @@
+import { useState } from 'react';
+import GameLayout from '../gameLayout/GameLayout';
+import QuarterBoard from '../quarterBoard/QuarterBoard';
+import Description from '../description/Description';
+
+function confirm(field, quarterNumber) {
+  // 5678 ABCD(1234) - 1
+  // 5678 EFGH(5678) - 2
+  // 1234 ABCD(1234) - 3
+  // 1234 EFGH(5678) - 4
+  const letters = 'ABCDEFGH';
+  const numbers = '12345678';
+  if (
+    (letters.slice(0, 4).includes(field[0]) === true)
+    && (numbers.slice(4, 8).includes(field[1]) === true)
+  ) {
+    if (quarterNumber === '1') {
```

Comment:
maybe moving to 
switch (quarterNumber) {
case '1':
and so on
will make more readable code. 
However, this requires unit tests to check if functionality is not changed.